codex-rs/mcp-server/src/lib.rs

```diff
@@ -77,10 +82,26 @@ pub async fn run_main(codex_linux_sandbox_exe: Option<PathBuf>) -> IoResult<()>
         }
     });
 
+    // Parse CLI overrides once so we can reuse them for operations
+    // like `get_auth_status` that need access to the effective config.
+    let cli_kv_overrides = match cli_config_overrides.parse_overrides() {
+        Ok(v) => v,
+        Err(e) => {
+            return Err(std::io::Error::new(
```

Comment:
Prefer importing stdlib types at the top, though admittedly something common like `Error` might need to be fully-qualified inline (but `ErrorKind` is less of an issue).